{"name": "test_sub[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1.0 == 1.2", "trace": "self = <calculator.testcase.test_calc_case.TestCalc object at 0x000001B3853538B0>, get_calc = <calculator.python_code.calc.Calculator object at 0x000001B385325A00>\nget_sub_datas = [2.2, 1.2, 1.2]\n\n    @allure.story(\"减法测试案例\")\n    @pytest.mark.run(order=2)\n    # @pytest.mark.second\n    def test_sub(self, get_calc, get_sub_datas):\n        \"\"\"\n        get_calc实例化计算器类\n        get_sub_datas:获取测试数据\n        \"\"\"\n        with allure.step(\"计算两个数的余\"):\n            # 获取yaml文件内参数\n            result = get_calc.sub(get_sub_datas[0], get_sub_datas[1])\n            # 如果相加数为float型 ，保留两位小数\n        if isinstance(result, float):\n            result = round(result, 2)\n            # 结果添加断言\n>       assert result == get_sub_datas[2]\nE       assert 1.0 == 1.2\n\ntest_calc_case.py:65: AssertionError"}, "description": "\n        get_calc实例化计算器类\n        get_sub_datas:获取测试数据\n        ", "steps": [{"name": "计算两个数的余", "status": "passed", "start": 1612516764501, "stop": 1612516764501}], "attachments": [{"name": "stdout", "source": "6115e9a3-a2f3-4646-8fa4-d3026186ae64-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_sub_datas", "value": "[2.2, 1.2, 1.2]"}], "start": 1612516764501, "stop": 1612516764502, "uuid": "80a5cdf6-0cd9-4d91-b122-129d944714de", "historyId": "e4585824fe38b3f22b0c1d04fab857f3", "testCaseId": "44542c927a1779c3d8e2b3ace4e424dd", "fullName": "test_calc_case.TestCalc#test_sub", "labels": [{"name": "feature", "value": "测试计算器类"}, {"name": "story", "value": "减法测试案例"}, {"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "test_calc_case"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-DH8SI2H"}, {"name": "thread", "value": "32452-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_case"}]}